adam/language_specific/chinese/chinese_language_generator.py

```diff
@@ -0,0 +1,477 @@
+                self.dependency_graph.add_edge(de, possessor, role=CASE_POSSESSIVE)
+                self.dependency_graph.add_edge(
+                    possessor, noun_dependency_node, role=NOMINAL_MODIFIER_POSSESSIVE
+                )
+            # if the count is one, we're done since we're not using yi CLF currently
+            if count == 1:
+                return
+            #  https://github.com/isi-vista/adam/issues/782
+            # TODO: get classifiers checked by a native speaker upon implementation
+            elif count == 2:
+                raise NotImplementedError(
+                    "We don't know how to handle Chinese classifiers yet"
+                )
+            # if the count is many, we don't need a CLF
+            else:
+                many = DependencyTreeToken("hen3 dwo1", NUMERAL)
+                self.dependency_graph.add_edge(
+                    many, noun_dependency_node, role=NUMERIC_MODIFIER
+                )
+
+        """Translate relations that the user explicitly calls out"""
+
+        def _translate_relation(
+            self,
+            action: Optional[Action[OntologyNode, SituationObject]],
+            relation: Relation[SituationObject],
+        ):
+            if relation.relation_type == HAS:
+                # if the situation is dynamic, then this will be handled within the NP
+                if (
+                    self.situation.is_dynamic
+                    or IGNORE_HAS_AS_VERB in self.situation.syntax_hints
+                ):
+                    pass
+                else:
+                    raise NotImplementedError
+            elif relation.relation_type == IN_REGION:
+                prepositional_modifier = self.relation_to_prepositional_modifier(
+                    action, relation
+                )
+                if prepositional_modifier:
+                    self.dependency_graph.add_edge(
+                        prepositional_modifier,
+                        self._noun_for_object(relation.first_slot),
+                        role=NOMINAL_MODIFIER,
+                    )
+            else:
+                raise RuntimeError(
+                    "We can't currently translate {} relation to Chinese".format(relation)
+                )
+
+        """Translate a relation to a prepositional modifier"""
+
+        def relation_to_prepositional_modifier(
+            self,
+            action: Optional[Action[OntologyNode, SituationObject]],
+            relation: Relation[SituationObject],
+        ) -> Optional[DependencyTreeToken]:
+
+            region = cast(SituationRegion, relation.second_slot)
+            # if the object in the relation is not salient, then we don't care about the relation
+            if region.reference_object not in self.situation.salient_objects:
+                return None
+            # deal with actions with verbs
+            if action:
+                raise NotImplementedError
```

Comment:
```suggestion
                raise NotImplementedError()
```

and preferably add a link to an issue to the message.